### PR TITLE
fix(STREETCO): NAV-2466

### PR DESCRIPTION
**Pull request by kodflow**
- minor fix

### CODE REVIEW

**Review by xlqian — APPROVED** [review]

**Review by azime — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 1]
```diff
@@ -188,27 +188,27 @@ def wrapper(*args, **kwargs):
...
+
+                if 'sections' in journey and 'region' in kwargs:
+                    args = request.args.to_dict(flat=False)
+                    args['region'] = kwargs['region']
+                    del args["from"]
+                    del args["to"]
+                    if 'data_freshness' in args:
+                        del args["data_freshness"]
+                    if 'datetime' in args:
+                        del args["datetime"]
```
Comment (azime) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: Minor:

```suggestion
                    for param in ["from", "to", "data_freshness", "datetime"]:
                        if param in args:
                            del args[param]
```

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 2]
```diff
@@ -188,27 +188,24 @@ def wrapper(*args, **kwargs):
...
-
-                            obstacle = create_external_link('v1.obstacles_nearby', **args)
-                            section['links'].append(obstacle)
+
+                if 'sections' in journey and 'region' in kwargs:
+                    args = request.args.to_dict(flat=False)
+                    args['region'] = kwargs['region']
+                   for param in ["from", "to", "data_freshness", "datetime"]:
+                   if param in args:
+                       del args[param]
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]:
```suggestion
                    for param in ["from", "to", "data_freshness", "datetime"]:
                    if param in args:
                        del args[param]
```

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 3]
```diff
@@ -188,27 +188,24 @@ def wrapper(*args, **kwargs):
...
-
-                            obstacle = create_external_link('v1.obstacles_nearby', **args)
-                            section['links'].append(obstacle)
+
+                if 'sections' in journey and 'region' in kwargs:
+                    args = request.args.to_dict(flat=False)
+                    args['region'] = kwargs['region']
+                    for param in ["from", "to", "data_freshness", "datetime"]:
+                    if param in args:
+                        del args[param]
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]:
```suggestion
                       if param in args:
                            del args[param]
```

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 4]
```diff
@@ -188,27 +188,24 @@ def wrapper(*args, **kwargs):
...
-
-                            obstacle = create_external_link('v1.obstacles_nearby', **args)
-                            section['links'].append(obstacle)
+
+                if 'sections' in journey and 'region' in kwargs:
+                    args = request.args.to_dict(flat=False)
+                    args['region'] = kwargs['region']
+                    for param in ["from", "to", "data_freshness", "datetime"]:
+                       if param in args:
+                            del args[param]
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]:
```suggestion
                        if param in args:
                            del args[param]
```

**Pull request merged**